doc/reference/configuration.rst

```diff
@@ -774,6 +774,21 @@ WorseReflectionExtension
 ------------------------
 
 
+.. _param_language_server_code_transform.import_globals:
```

Comment:
will move all the diagnostics to a dedicated package and rename this accordingly. leaving as is for now for b/c